src/helpers.php

```diff
@@ -47,6 +47,6 @@ function global_hook_dir()
      */
     function absolute_git_dir()
     {
-        return trim(shell_exec('git rev-parse --absolute-git-dir'));
+        return trim(shell_exec('git rev-parse --git-dir'));
```

Comment:
the problem is `--git-dir` resolves to `.git` if in the same directory. you need to add `realpath(trim(shell_...`

Reply:
@BrainMaestro Yea i missed that, now it is working